datadog/resource_datadog_logs_metric_test.go

```diff
@@ -0,0 +1,122 @@
+package datadog
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccDatadogLogsMetric_Basic(t *testing.T) {
+	accProviders, clock, cleanup := testAccProviders(t, initRecorder(t))
+	uniqueLogsMetric := strings.ReplaceAll(uniqueEntityName(clock, t), "-", "_")
+	defer cleanup(t)
+	accProvider := testAccProvider(t, accProviders)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    accProviders,
+		CheckDestroy: testAccCheckDatadogLogsMetricDestroy(accProvider),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckDatadogLogsMetricConfig_Basic(uniqueLogsMetric),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckDatadogLogsMetricExists(accProvider, "datadog_logs_metric.testing_logs_metric"),
+					resource.TestCheckResourceAttr(
+						"datadog_logs_metric.testing_logs_metric", "compute.aggregation_type", "distribution"),
+					resource.TestCheckResourceAttr(
+						"datadog_logs_metric.testing_logs_metric", "compute.path", "@duration"),
+					resource.TestCheckResourceAttr(
+						"datadog_logs_metric.testing_logs_metric", "filter.query", "service:test"),
+					resource.TestCheckResourceAttr(
+						"datadog_logs_metric.testing_logs_metric", "group_by.#", "2"),
+					resource.TestCheckResourceAttr(
+						"datadog_logs_metric.testing_logs_metric", "group_by.0.path", "@my.status"),
+					resource.TestCheckResourceAttr(
+						"datadog_logs_metric.testing_logs_metric", "group_by.0.tag_name", "status"),
+					resource.TestCheckResourceAttr(
+						"datadog_logs_metric.testing_logs_metric", "group_by.1.path", "service"),
+				),
+			},
+		},
+	})
+}
+
+func testAccCheckDatadogLogsMetricConfig_Basic(uniq string) string {
+	return fmt.Sprintf(`
+        resource "datadog_logs_metric" "testing_logs_metric" {
+			name = "%s"
+			compute = {
+				aggregation_type = "distribution"
+				path             = "@duration"
+			}
+			filter = {
+				query = "service:test"
+			}
+			group_by {
+				path     = "@my.status"
+				tag_name = "status"
+			}
+			group_by {
+				path     = "service"
+				tag_name = "service"
+			}
+        }
+    `, uniq)
+}
+
+func testAccCheckDatadogLogsMetricExists(accProvider *schema.Provider, resourceName string) resource.TestCheckFunc {
```

Comment:
We just recently removed our usage of Exists since its redundant in the terraform sdk and was deprecated upstream. Can you remove this method and check for a 404 in the Read method and set ID to "" if it doesn't exist there?

More context/examples here - https://github.com/DataDog/terraform-provider-datadog/pull/805